doxia-modules/doxia-module-apt/src/main/java/org/apache/maven/doxia/module/apt/AptSink.java

```diff
@@ -817,6 +820,9 @@ public void text(String text, SinkEventAttributes attributes) {
         } else {
             content(text);
         }
+        if (attributes != null) {
+            inline_();
+        }
```

Comment:
Is that a bug we haven't noticed before?

Reply:
Yes, similar to https://issues.apache.org/jira/browse/DOXIA-706

Reply:
Shouldn't this be separate then?

Reply:
Ok -> https://issues.apache.org/jira/browse/DOXIA-715